pkg/testutil/util.go

```diff
@@ -79,11 +80,16 @@ func IsYamlEqual(expectation *string, actualYamlByteArray *[]byte) bool {
 	}
 
 	// Replace Timestamps that would show up as different.
-	_, err := exec.Command("sed", "-r", "-i", ReplaceTimestampRegExp, actualYamlFileName).Output()
+	_, err := exec.Command("sed", "-r", "-i.tmp", ReplaceTimestampRegExp, actualYamlFileName).Output()
```

Comment:
Why are we using command line arguments within Golang to begin with? Go has support for marshaling and unmarshaling YAML into structs, and Go supports determining differences between structs. Especially when deleting or reading files (the `cat` lower down), there is no reason NOT to use the `os` package.

Reply:
This is just a temp fix to get unit tests working on MACOSX, the unit test code here can be rewritten in a separate pr.